edsnlp/pipelines/misc/sections/patterns.py

```diff
@@ -68,6 +72,7 @@
     r"examens complementaires realises pendant le sejour",
     r"examens para-cliniques",
     r"imagerie post-operatoire",
+    r"examens complementaires (Résultats et commentaires)"
```

Comment:
Lowercase + accent stripping since we expect `eds.normalizer` to be run before

```suggestion
    r"examens complementaires (resultats et commentaires)"
```